### PR TITLE
Personal apps 

### PR DESCRIPTION
**Pull request by clearab**
Splitting https://github.com/microsoftgraph/microsoft-graph-docs/pull/5089/files?file-filters%5B%5D= to isolate private preview changes from beta changes.

### CODE REVIEW

File: api-reference/beta/api/user-add-teamsappinstallation.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |  User.ReadWrite.All, Directory.ReadWrite.All     |
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | User.ReadWrite.All, Directory.ReadWrite.All |
+
+## HTTP request
+
```
Comment (Lauragra) [inline api-reference/beta/api/user-add-teamsappinstallation.md]: Global: We need HTML comments for all HTTP requests and request and response examples. Looks like some of these files are missing them in some places.

File: concepts/changelog.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 | :-------------- | :------------ | :--------------------------------------- |
 | Addition | beta | Added support for application permissions to [List channel messages](/api-reference/beta/api/channel-list-messages.md), [Get channel message](/api-reference/beta/api/channel-get-message.md), [List replies to a message](/api-reference/beta/api/channel-list-messagereplies.md), and [Get a reply to a message](/api-reference/beta/api/channel-get-messagereply.md). |
 | Addition | beta | Added support for application permissions to [List messages in a chat](/api-reference/beta/api/chatmessage-list.md) and [Get message in chat](/api-reference/beta/api/chatmessage-get.md). |
+| Addition | beta | Added the **installedApps** property to the [chat](/api-reference/beta/resources/chat.md) resource.|
+| Addition | beta | Added navigation binding for **chats** property of the [user](/api-reference/beta/resources/user.md) resource.|
+| Addition | beta | Added the [teamwork](/api-reference/beta/resources/teamwork.md) resource and associated methods. |
```
Comment (Lauragra) [inline concepts/changelog.md]: The teamwork and userTeamwork resources don't have associated methods, right? Pls. review the changelog entry; you might want to call out the new operations. Can you describe at a high level the purpose for the new APIs from the POV of the developer?

File: api-reference/beta/toc.yml [hunk 3]
```diff
@@ -9255,6 +9258,26 @@
...
+      - name: Apps
+        href: resources/teamsappinstallation.md
+        items:
+        - name: List apps in team
+          href: api/teamsappinstallation-list.md
+        - name: Add app to team
+          href: api/teamsappinstallation-add.md
+        - name: Remove app from team
+          href: api/teamsappinstallation-delete.md
+        - name: List apps for a user
```
Comment (Lauragra) [inline api-reference/beta/toc.yml]: For the user-related methods, it seems that we should have a Teamwork node under User in the TOC, and surface these there as well as here. This is similar to how we expose other services under Users.

Comment (clearab) [reply api-reference/beta/toc.yml]: added

File: api-reference/beta/resources/teamsappinstallation.md [hunk 4]
```diff
@@ -10,23 +11,26 @@ ms.prod: "microsoft-teams"
...
 
 ## Methods
 
 | Method       | Return Type  |Description|
 |:---------------|:--------|:----------|
-|[List apps](../api/teamsappinstallation-list.md) | [teamsAppInstallation](teamsapp.md) | Lists apps installed in a team.|
-|[Add app](../api/teamsappinstallation-add.md) | [teamsAppInstallation](teamsapp.md) | Adds (installs) an app to a team.|
-|[Remove app](../api/teamsappinstallation-delete.md) | None | Removes (uninstalls) an app from a team.|
-|[Upgrade app](../api/teamsappinstallation-upgrade.md) | None | Upgrades to the latest version of the app.|
+|[List apps installed in team](../api/teamsappinstallation-list.md) | [teamsAppInstallation](teamsapp.md) | Lists apps installed in a team.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/teamsappinstallation.md]: @clearab , I am looking at your PR to identify why you were getting API Doctor errors for your examples. In the course of doing that, I noticed some related issues - can you confirm if line 20, Return Type column, teamsAppInstallation should indeed link to teamsappinstallation.md, and not teamsapp.md? It's ambiguous there.

BTW, the correct response type matters when I am fixing the HTML for the example responses.

Comment (clearab) [reply api-reference/beta/resources/teamsappinstallation.md]: Updated, and also added "collection" as I believe that is appropriate for this return type?

**Comment by clearab** [comment]
@Lauragra I believe this is good to go now, correct?

**Comment by Lauragra** [comment]
@clearab , merged. Will go live at EOD today. If you want to publish sooner, LMK.

**Pull request merged**